### DISCUSSION TITLE
[Unstable] An outpost deathmatch crate spawnpoint is outside

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#14480] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/user-attachments/assets/55958e8e-e555-49fc-8876-d2cd70681299)

![image](https://github.com/user-attachments/assets/b20aa964-be31-4f13-9f2b-8d4e636f7da8)

### Reproduction steps

1. Launch outpost deathmatch
2. Wait till unobtainable crate spawns

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/4162891cd6fe255a502d8e7cd25154f8277dbeb8
